spec/runners/transactions.rb

```diff
@@ -40,7 +40,7 @@ def define_transactions_spec_tests(test_paths)
                 elsif !test.multiple_mongoses? && SpecConfig.instance.addresses.length > 1
                   # Many transaction spec tests that do not specifically deal with
                   # sharded transactions fail when run against a multi-mongos cluster
-                  skip "Test does not specify multiple mongoses"
+                  skip "Test does not specify multiple mongoses" unless Utils.serverless?
```

Comment:
I am having trouble parsing this. Can the new condition be moved to the if-elsif level and a note added reflecting what the behavior should be under serverless and why?

Reply:
Now I think that this additional `unless` is not necessary. We should treat serverless instances as multiple mongoses environment without a special case here.
So, the check is removed.

Reply:
In theory any tests that succeed against  2 mongoses should also succeed against 3 mongoses, and if they don't that is probably a bug.